lib/services.cf

```diff
@@ -171,15 +174,17 @@ bundle agent standard_services(service,state)
 
       "have_init" expression => fileexists($(init));
 
-    chkconfig.have_init.freebsd::
-      "running" -> { "CFE-3513" }
-        expression => returnszero("$(init) onestatus > /dev/null", "useshell");
-
-    chkconfig.have_init.!freebsd::
+    chkconfig.have_init::
```

Comment:
As previously discussed in https://github.com/cfengine/masterfiles/pull/1892 checkconfig does not relate to freebsd, so this was removed for clarity.